Azure/sonic-mgmt
# Vnet vxlan pytest conversion

### Description of PR


Summary:
Fixes # (issue)

### Type of change




### Approach
#### What is the motivation for this PR?
Migrate VNET VxLAN test to pytest framework
#### How did you do it?
- Create test module for VNET VxLAN
- Copy necessary Jinja templates to pytest-specific template directory

#### How did you verify/test it?
- Run pytest and check that all cases (Disabled, Enabled, Cleanup) succeed
- Run with scaled options (num_vnet, num_routes, num_endpoints) and check that the given values are reflected in the generated config files 
- Run with `--skip_cleanup` and verify that cleanup step is skipped
- Run with `--ipv6_vxlan_test` and verify correct values in generated config files

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation 

Run this test with:

```
./run_tests.sh -i <inventory file> -d <DUT name> -n <testbed name> -u -c vxlan/test_vnet_vxlan.py
```


Use extra CLI parameters as following:
```
./run_tests.sh ...(same as above)... -e "--param_with_value=<value> --param_without_value"
```

Scaled VNET test:
```
--num_vnet=<x>
--num_routes=<y>
--num_endpoints=<z>
```

Other:
```
--skip_cleanup (don't remove VNET info after test ends)
```
